src/API/WP/OAuthService.php

```diff
@@ -159,16 +159,73 @@ public function revoke_wpcom_api_auth(): string {
 		$request = $this->container->get( Jetpack::class )->remote_request( $args );
 
 		if ( is_wp_error( $request ) ) {
+
+			/**
+			 * When the WPCOM token has been revoked with errors.
+			 *
+			 * @event revoke_wpcom_api_authorization
+			 * @property string status The status of the request.
+			 * @property string error The error message.
+			 * @property mixed blog_id The blog ID.
```

Comment:
Instead of mixed WDYT about `number|null` ?

Reply:
Also, maybe we can consider sending 400 or 500 for example or get the error code to have consistency between both `woocommerce_gla_track_event` tracks (in one is number and in other string "error" )

Reply:
error seems to be optional